### PR TITLE
Apply Cycle4 to main

### PR DESCRIPTION
**Pull request by h8718**
## Description

Please describe the aim of the pull request and the changes made in the commits

## Checklist

- [ ] Documentation updated if required
- [ ] Test coverage is appropriate
      
## Checklist Internal

- [ ] Linear issue linked (e.g. [DIR-XXXX] pull request title)
- [ ] Has the PR been labeled

### CODE REVIEW

**Comment by h8718** [comment]
We can merge this PR after the release

File: pkg/refactor/service/helper.go [hunk 1]
```diff
@@ -283,20 +284,32 @@ func buildEnvVars(withGrpc bool, c *core.Config, sv *core.ServiceFileData) []cor
...
+		})
 	}
 
 	if len(c.KnativeProxyHTTPS) > 0 {
 		proxyEnvs = append(proxyEnvs, corev1.EnvVar{
 			Name:  util.DirektivProxyHTTPS,
 			Value: c.KnativeProxyHTTPS,
 		})
+		proxyEnvs = append(proxyEnvs, corev1.EnvVar{
+			Name:  strings.ToLower(util.DirektivProxyHTTPS),
```
Comment (sir-hassan) [inline pkg/refactor/service/helper.go:299]: @jensg-st , What does it means to lower-case an env var name?

Comment (jensg-st) [reply pkg/refactor/service/helper.go:299]: Python needs lowercase other things need uppercase.

Comment (sir-hassan) [reply pkg/refactor/service/helper.go:299]: Oh I see.

**Pull request merged**